ci/release/update-version.sh

```diff
@@ -85,8 +85,11 @@ sed_runner "s/extra-repo-sha: branch-.*/extra-repo-sha: branch-${NEXT_SHORT_TAG}
 # Wheel builds install dask-cuda from source, update its branch
 sed_runner "s/dask-cuda.git@branch-[^\"\s]\+/dask-cuda.git@branch-${NEXT_SHORT_TAG}/g" .github/workflows/*.yaml
 
+CURRENT_SHORT_TAG_PEP440=$(python -c "from setuptools.extern import packaging; print(packaging.version.Version('${CURRENT_SHORT_TAG}')")
+NEXT_SHORT_TAG_PEP440=$(python -c "from setuptools.extern import packaging; print(packaging.version.Version('${NEXT_SHORT_TAG}')")
+
 # Wheel builds install intra-RAPIDS dependencies from same release
-sed_runner "s/${CURRENT_SHORT_TAG}/${NEXT_SHORT_TAG}/g" python/cugraph/setup.py
-sed_runner "s/${CURRENT_SHORT_TAG}/${NEXT_SHORT_TAG}/g" python/cugraph/_custom_build/backend.py
-sed_runner "s/${CURRENT_SHORT_TAG}/${NEXT_SHORT_TAG}/g" python/pylibcugraph/setup.py
-sed_runner "s/${CURRENT_SHORT_TAG}/${NEXT_SHORT_TAG}/g" python/pylibcugraph/_custom_build/backend.py
+sed_runner "s/${CURRENT_SHORT_TAG_PEP440}/${NEXT_SHORT_TAG_PEP440}/g" python/cugraph/setup.py
+sed_runner "s/${CURRENT_SHORT_TAG_PEP440}/${NEXT_SHORT_TAG_PEP440}/g" python/cugraph/_custom_build/backend.py
+sed_runner "s/${CURRENT_SHORT_TAG_PEP440}/${NEXT_SHORT_TAG_PEP440}/g" python/pylibcugraph/setup.py
+sed_runner "s/${CURRENT_SHORT_TAG_PEP440}/${NEXT_SHORT_TAG_PEP440}/g" python/pylibcugraph/_custom_build/backend.py
```

Comment:
cross-posting the comment below since it's relevant here as well

- https://github.com/rapidsai/raft/pull/1200#discussion_r1089343751